int8 joint_id
int32 pulse
float32 radians
float32 degrees
